brainsets_pipelines/peterson_brunton_pose_trajectory_2022/pipeline.py

```
@@ -0,0 +1,706 @@
+
+    def _generate_splits(
+        self,
+        subject_id: str,
+        session_id: str,
+        active_behavior_trials: Interval,
+        active_vs_inactive_trials: Interval,
+        pose_valid_domain: Interval,
+    ) -> Data:
+        # Uses nested splits.{task}_fold_{k}_{split} keys rather than the
+        # set_train_domain/set_valid_domain/set_test_domain convention because
+        # this dataset has multiple tasks × multiple folds.
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)